packages/cli/src/ceramic-daemon.ts

```diff
@@ -816,6 +816,7 @@ export class CeramicDaemon {
     const streamId = StreamID.fromString(req.params.streamid || req.params.docid)
     const { force } = req.body
     await this.ceramic.pin.add(streamId, force)
+    Metrics.count(METRIC_NAMES.STREAM_PINNED, 1)
```

Comment:
Some of these flows are synchronous API calls, will the metrics code add any latency?

Reply:
it should not - it should be counting internally and the exporter able to be scraped.  it does not have to do a network call to count.